src/lib/components/cards/ContentIntro.svelte

```diff
@@ -103,6 +103,18 @@
 				{/if}
 			{/if}
 		</div>
+		{#if overview.contentType === ContentTypeEnum.Course}
+			<div class="author-wrapper">
+				<a
+					href={overview.author?.authorLink}
+					target="_blank"
+					rel="noopener noreferrer"
+					class="header-link row-2 align-center"
+				>
+					<span>Author: {overview.author?.authorName}</span>
```

Comment:
@IgnacioDebat <Button color="neutral" type="transparent"> will look much better.
Also, with the prop iconLeft we can put an icon here, maybe "tabler:pencil" is a good option?

Reply:
It'll also be good to wrap this new author block inside an {#if} wrapper so if the course doesn't have an author it doesn't break

Reply:
Chino super small adjustment here...
I'll take out some spacing between both buttons (author and edit content) and add more gap between the TableOfContent and the buttons..

Reply:
![image](https://user-images.githubusercontent.com/112862961/232861140-74ec2612-cb47-4f0f-b3b9-4b8eb6233a5e.png)

Something like this?